tests/sentry/utils/performance_issues/test_performance_detection.py

```diff
@@ -129,22 +129,6 @@ def test_project_option_overrides_default(self):
         perf_problems = _detect_performance_problems(n_plus_one_event, sdk_span_mock, self.project)
         assert perf_problems == []
 
-    # TODO Abdullah Khan: Uncomment after detection_rate migration
-    # @override_options(BASE_DETECTOR_OPTIONS)
-    # def test_detection_enable_option_passed_to_detector(self):
-    #     n_plus_one_event = get_event("n-plus-one-in-django-index-view")
-    #     sdk_span_mock = Mock()
-
-    #     perf_problems = _detect_performance_problems(n_plus_one_event, sdk_span_mock, self.project)
-    #     assert_n_plus_one_db_problem(perf_problems)
-
-    #     self.project_option_mock.return_value = {
-    #         "n_plus_one_db_queries_detection_enabled": False,
-    #     }
-
-    #     perf_problems = _detect_performance_problems(n_plus_one_event, sdk_span_mock, self.project)
-    #     assert perf_problems == []
```

Comment:
Did you mean to remove this test?

Reply:
Yeah, I mentioned it the PR description. It's an overlapping test. Will add a better test in the coming PRs.

Reply:
Oops, I missed that! Thanks for clarifying